tests/cases/whitespace/use-namespace

```diff
@@ -1,16 +0,0 @@
----DESCRIPTION---
-Use declarations must go after the namespace declaration
----CONTENTS---
-<?php
-
-declare(strict_types=1);
-
-use Foo\Bar;
-
-namespace Vendor;
-
-new Bar();
-
----MESSAGES---
-5:1 PSR2.Namespaces.UseDeclaration.UseBeforeNamespace
```

Comment:
`SlevomatCodingStandard.Namespaces.UnusedUses` now removes the use statements, so we can forget this.